packages/muelu/src/Transfers/Smoothed-Aggregation/MueLu_NullspaceFactory_def.hpp

```diff
@@ -93,6 +98,13 @@ namespace MueLu {
     if (currentLevel.IsAvailable(nspName, NoFactory::get()) == false && currentLevel.GetLevelID() == 0)
       Input(currentLevel, "A");
 
+    if( currentLevel.GetLevelID() == 0 &&
+        currentLevel.IsAvailable("Coordinates", NoFactory::get()) &&     // we have coordinates (provided by user app)
+        pL.get<bool>("nullspace: calculate rotations") ) {               // and we want to calculate rotation modes
+      calculateRotations_ = true;
+      Input(currentLevel, "Coordinates");
```

Comment:
Are we sure this Input declaration is correct?

Reply:
@csiefer2 it has definitely been tested. Is there something funny or strange about it that makes you concerned?